packages/netlify-cms-core/src/__tests__/backend.spec.js

```diff
@@ -131,14 +130,8 @@ describe('Backend', () => {
       const implementation = {
         init: jest.fn(() => implementation),
       };
-      const config = {
-        ...defaultConfig,
-        backend: {
-          name: 'github',
-        },
-      };
 
-      const backend = new Backend(implementation, { config, backendName: config.backend.name });
+      const backend = new Backend(implementation, { config: {}, backendName: 'github' });
```

Comment:
@erezrokah why do you think it's better to not use real default state? When tests are converted to typescript we'll have to either put a lot of tsignore statements or "as SomeType" statements.

Reply:
I worried this will make the tests brittle. Personally I like to have in the tests as little as information possible required for them to pass so I don't have to change them too often.

As for typing the tests - we'll have to see if that makes our lives easier or not? I would expect tests to fail when something breaks and not rely on types?

Reply:
> I would expect tests to fail when something breaks and not rely on types
this is true. On the other hand you're basically passing incorrect values to functions being tested. So tests might pass because of that. And might fail if correct values are provided.

Reply:
Alright